DMTN-165.tex

```diff
@@ -50,7 +56,289 @@
+Additionally it requires brokers to re-engineer existing alert retrieval code.
+The handful of full-stream brokers may view these as pure costs with no benefit relative to the baseline.
+We believe these costs are justified by the gains of being able to distribute the alerts more widely; the modest additional latency is not scientifically significant.
+As discussed above, brokers may ultimately find it technically advantageous to be able to have their own systems direct and parallelize the retrieval of the largest volumes of data.
+
+%  Data rights issues
+The most pressing concerns are those of data access.
+Alert packet contents are world public; this would include both the lightweight alerts and the full-sized alerts.
+However, access to databases and services within the Rubin Data Facility is restricted to data-rights holders \citedsp{RDO-013}.
+If the same consideration applies here, only users with data rights could subscribe to the lightweight stream and access the full-sized alerts directly\footnote{We do support investigating whether true world-public \textit{access} to the lightweight alerts and full alerts with very low rate limits could be supported.}.
+As in the current baseline, users without data rights would rely on community alert brokers to access the public alerts.
+Since the lightweight alerts could be much more widely distributed to individual users, this situation would highlight more starkly the distinction between data rights and data access to the (world-public) alerts, which might create confusion or frustration among scientists.
+
+%need for authentication (for data rights, also for rate limits)
+Both the lightweight alert stream and the packet retrieval service would require authentication and authorization to confirm data rights and set appropriate rate limits.
+Individuals may find rate limits frustrating if their alert retrieval needs don't match their rate limits and may pressure the project for greater access rather than adopting their workflow to use a community broker.
+% is this different than the baseline?
+
+The lightweight alerts contain a URI that points to the corresponding full alert packet.
+If a user without data rights is unable to access the packet at that location, having a copy of the world-public lightweight alert is not very useful.
+While community brokers are responsible for providing public access to alerts even in the baseline, they would need to provide their own additional URL or URI pointing to a publicly-accessible copy of the full alert, which could create confusion.
+From a pure user-experience perspective this suggests making access to the full alerts world-public, even if the rate limits for non-data-rights holders are extremely low.
+
+Referential integrity of the lightweight and full alerts could be hard to maintain. 
+Perfect synchronization of the data and notification systems is probably impossible, which could impact users' ability to rapidly retrieve the full alerts.
```

Comment:
Perfect synchronization may be difficult, but ensuring that the full alert is present before publishing a lightweight alert is somewhat easier to guarantee (i.e. the lightweight stream may miss some but may not publish extra).